etc/config/c++.amazon.properties

```diff
@@ -213,14 +213,26 @@ group.cross.groupName=Cross GCC
 ###############################
 # GCC for PPC
 group.ppc.compilers=ppcg48:ppc64leg630
-group.ppc.groupName=PowerPC GCC
+group.ppc.groupName=POWER Compilers
 group.ppc.isSemVer=true
 compiler.ppcg48.exe=/opt/compiler-explorer/powerpc/gcc-4.8.5/bin/powerpc-unknown-linux-gnu-g++
 compiler.ppcg48.name=PowerPC gcc 4.8.5
 compiler.ppcg48.semver=4.8.5
 compiler.ppc64leg630.exe=/opt/compiler-explorer/powerpc64le/gcc-6.3.0/powerpc64le-unknown-linux-gnu/bin/powerpc64le-unknown-linux-gnu-g++
-compiler.ppc64leg630.name=PowerPC64 gcc 6.3.0
+compiler.ppc64leg630.name=power64le gcc 6.3.0
 compiler.ppc64leg630.semver=6.3.0
+compiler.ppc64leg8.exe=/opt/compiler-explorer/powerpc64le/gcc-at12/powerpc64le-unknown-linux-gnu/bin/powerpc64le-unknown-linux-gnu-g++
+compiler.ppc64leg8.name=power64le at gcc 8 (trunk)
+compiler.ppc64leg8.semver=8
```

Comment:
This semver should be of the form `8.X.Y`

Reply:
thanks. Updated. "trunk" in the name wasn't really accurate so changed this too.